angr/exploration_techniques/crash_monitor.py

```diff
@@ -0,0 +1,185 @@
+import angr
```

Comment:
I think that instead of importing `angr` and doing `angr.exploration_techniques.CrashMonitor`, you can just reference `CrashMonitor` everywhere below... That's also cleaner import-wise.